engine/classes/ElggUser.php

```diff
@@ -183,6 +183,23 @@ protected function update() {
 	}
 
 	/**
+	 * Get user language or default to site language
+	 *
+	 * @param string $fallback Optional language
```

Comment:
Instead of `Optional language`, how about `If this is provided, it will be returned if the user doesn't have a language set. If null, the site language will be returned in that case.`